src/orders/views/OrderTransactionRefundCreate/OrderTransactionRefundCreate.tsx

```diff
@@ -46,7 +47,29 @@ const OrderTransactionRefund: React.FC<OrderTransactionRefundCreateProps> = ({
           ),
         );
       }
+      if (submitData?.orderGrantRefundCreate?.errors.length) {
```

Comment:
suggestion: above have if that checking if no errors, couldn't use else instead of new if